roles/devscripts/tasks/110_check_ocp.yml

```diff
@@ -14,27 +14,25 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-# Check for existing deployment
 
-
-- name: Determine if the role was run with cifmw_devscripts_use_layers.
+- name: Check for pre-existences of OpenShift cluster.
   ansible.builtin.find:
     file_type: "file"
     path: "{{ cifmw_devscripts_config.working_dir }}"
     patterns: "{{ cifmw_devscripts_config.cluster_name }}_*.xml"
   register: overlay_results
 
-- name: Using existing deployment layer for creating OCP cluster.
+- name: Verify the node definition match with expected count.
   when: >-
     (
       cifmw_devscripts_config.num_masters | int +
       cifmw_devscripts_config.num_workers | int
     ) == overlay_results.matched
   ansible.builtin.set_fact:
-    cifmw_devscripts_use_base_layer: true
+    cifmw_devscripts_ocp_exists: true
```

Comment:
you can use the `when` condition as the actual value here, allowing to remove the `|default(false)` maybe?

Reply:
Fixed in the next commit.